packages/govern-create/helpers/types.ts

```diff
@@ -0,0 +1,19 @@
+export type Address = string
+
+export enum eEthereumNetwork {
+  rinkeby = 'rinkeby',
+  main = 'main',
+  hardhat = 'hardhat',
+}
+
+export enum eContractid {
```

Comment:
I would recommend to define it like this ``const enum eContractId{...}`` the generated JS is much smaller.

I would also not name those like this.. just call it ``CONTRACT_ID`` or ``CONTRACT_NAMES``. ``eEthereumNetwork`` should for example also be called ``ETHEREUM_NETWORK``.

Reply:
I also don't think ``types.ts`` is the right place to define enum's cause they aren't just types :-)

Reply:
Agreed, I only learned recently about the difference between `enum` and `const enum` :smile:  

Also I don’t think we need string values?

Reply:
Agreed! :)